netci.groovy

```diff
@@ -13,13 +13,16 @@ def projectFolder = Utilities.getFolderName(project) + '/' + Utilities.getFolder
 // Globals
 
 // Map of osName -> osGroup.
-def osGroupMap = ['Ubuntu14.04':'Linux',
+def osGroupMap = ['Windows 10':'Windows_NT',
+                  'Windows 7':'Windows_NT',
```

Comment:
I see that osShortName for Windows 10 -> win10 , Windows 7 -> win7, and Windows Nano 2016 -> winnano16 

Should that be the case here, instead of Windows_NT for all three ? i.e Windows 10 -> Windows 10?